README.md

```diff
@@ -1348,6 +1348,7 @@ Update Time, five active automations, webhooks.
 ## Tunneling, WebRTC, Web Socket Servers and Other Routers
 
   * [Pinggy](https://pinggy.io) — Public URLs for localhost with a single command, no downloads required. HTTPS / TCP / TLS tunnels. The free plan has 60 minutes tunnel lifetime.
+  * [Beeceptor's Local Tunnel](https://beeceptor.com/local-tunnel/?utm_source=free-for-dev) — Download CLI, connect and browse! Share the localhost services globally in a snap! It supports HTTPS out of the box, works over web sockets, real-time dashboard to inspect requests, mock selective routes & request/response history. The free tier is available, with daily request limits.
```

Comment:
50 requests a day isn't useful sorry, that's like one page load of a medium size website.

Reply:
That's true with modern SPA apps. Hosting/testing websites from localhost isn't just one use-case for tunnels. 

Many developers use tunnels for: 
- webhook development and testing
- API integration testing (between services or frontend+backend)
- Mobile app integration testing.

The free limit is a daily reset. There are no limits on how many tunnels can be created. Also, it doesn't even require any signup. The paid plans for Beeceptor are small and affordable.

Reply:
Yeah I hear you but sorry its too low so I won't be accepting this one as is.

Reply:
No problem, you can close this PR. It's your list and opinionated.
besides, kudos to your work to compile a comprehensive list for the community. 👏

Reply:
Thank you